### PR TITLE
Add log_flags to hekad config

### PR DESCRIPTION
**Pull request by sathieu**

### CODE REVIEW

**Comment by sathieu** [comment]
Complement #1742 for all the non-decoder errors.

Context: I need this because, under systemd, all stdout/stderr is logged (and forwarded to syslog, the timestamp is printed twice)

File: docs/source/config/index.rst [hunk 1]
```diff
@@ -164,6 +164,13 @@ Config:
     The maximum size (in bytes) of message can be sent during processing.
     Defaults to 64KiB.
 
+.. versionadded:: 0.10
+
+- log_flags (int):
+    Control the prefix for STDOUT logs. Does not control the `LogOutput` plugin.
```
Comment (rafrombrc) [inline docs/source/config/index.rst]: STDOUT and STDERR

Comment (rafrombrc) [inline docs/source/config/index.rst]: Also, was there a reason why you didn't propagate the behaviour down to the LogOutput?

**Comment by rafrombrc** [comment]
Needs changelog entry. :)

File: docs/source/config/index.rst (hunk 1 above)
Comment (sathieu) [inline docs/source/config/index.rst]: No. Just that I don't need it.

i'll try to improve the patch.

**Pull request merged**